### ISSUE TITLE
new account -> never joined any server (yet) -> remote host is closing websocket connection

### ISSUE THREAD

**Issue by 0x17de** [sm00th/bitlbee-discord#66]
I had a new discord account and never joined any server before. The resulting issue was the following over and over again:
```
<root> discord - Logging in: Remote host is closing websocket connection
<root> discord - Logging in: Signing off..
<root> discord - Logging in: Reconnecting in 900 seconds..
```
After joining the first server on the discord webapp the issue was resolved.

**Comment by sm00th** [comment]
Yeah, in this case the initialization procedure as bitlbee-discord sees it would never finish.
I see it as a low-priority issue though as there is no way to add new servers in bitlbee-discord at the moment.

**Comment by Sophira** [comment]
I'm getting this error even though my Discord account has joined servers. It happens even after I've been able to connect and chat.

**Comment by sm00th** [comment]
Can you get debug logs (see https://wiki.bitlbee.org/Debugging) when this happens? Make sure to at least remove password/token info before posting the results.

**Comment by Sophira** [comment]
I took a look in my IRC logs and actually it seems that it was only a temporary thing - I only got that message between 12:08:21 and 12:19:16 GMT today. It's probably just Discord as I didn't get that message for several months prior. (I do get "Remote host is closing websocket connection" very commonly, but that's obviously a different issue.)

Sorry for the false alarm! If it does happen again I'll see if I can get debug logs but I don't think it's the plugin's fault in this case.
